src/GenAPI/Microsoft.DotNet.GenAPI/SyntaxRewriter/TypeForwardAttributeCSharpSyntaxRewriter.cs

```diff
@@ -10,33 +12,127 @@ namespace Microsoft.DotNet.GenAPI.SyntaxRewriter
     /// <summary>
     /// Handles type forward assembly attributes and removes generic type arguments:
     ///     [assembly:TypeForwardedToAttribute(typeof(System.Collections.Generic.IAsyncEnumerable<A, B, C>))] ->
-    ///     [assembly:TypeForwardedToAttribute(typeof(System.Collections.Generic.IAsyncEnumerable<,,>))] ->
+    ///     [assembly:TypeForwardedToAttribute(typeof(System.Collections.Generic.IAsyncEnumerable<,,>))]
+    ///
+    /// Also handles type forwards to Tuples
+    ///     [assembly:TypeForwardedToAttribute(typeof((T1, T2, T3))] ->
+    ///     [assembly:TypeForwardedToAttribute(typeof(System.ValueTuple<,,>))]
     /// </summary>
     public class TypeForwardAttributeCSharpSyntaxRewriter : CSharpSyntaxRewriter
```

Comment:
Recommendation for improving performance (in the future):
What if we modify the CSharpFileBuilder class so that we only run the TypeForwardAttributeCSharpSyntaxRewriter on generated type forwards [CSharpFileBuilder.cs#L91](https://github.com/dotnet/sdk/blob/main/src/GenAPI/Microsoft.DotNet.GenAPI/CSharpFileBuilder.cs#L318), instead of applying it to the entire generated file [CSharpFileBuilder.cs#L99](https://github.com/dotnet/sdk/blob/main/src/GenAPI/Microsoft.DotNet.GenAPI/CSharpFileBuilder.cs#L99)?

Reply:
I was able to remove this completely.